lib/dal/src/action.rs

```diff
@@ -1,3 +1,4 @@
+use petgraph::Direction;
```

Comment:
nit: feel free to ignore. The order or imports here is weird. I would not usually expect module declarations in our import ordering... at least within this repo.

Thoughts?